geode-connectors/src/main/java/org/apache/geode/connectors/jdbc/internal/cli/CreateDataSourceCommand.java

```diff
@@ -148,6 +149,15 @@ public boolean updateConfigForGroup(String group, CacheConfig config, Object ele
     return true;
   }
 
+  @CliAvailabilityIndicator({CREATE_DATA_SOURCE})
+  public boolean commandAvailable() {
+    boolean isAvailable = true;
```

Comment:
Instead of repeating this implementation multiple times just do this in the body of the method:
    return isOnlineCommandAvailable();

Reply:
Done!